labellab-client/src/components/project/css/labelItem.css

```diff
@@ -23,3 +23,18 @@
   padding: 3;
   font-size: 24;
 }
+@media only screen and (max-width: 892px){
```

Comment:
Use standard dimensions in media query like `max-width: 768px;`

Reply:
@Tushar19varshney the change needed is on that specific width because between 768px and 892px buttons already go out of the card, that's why i used that dimension.